loky/backend/semaphore_tracker.py

```diff
@@ -80,7 +90,8 @@ def ensure_running(self):
             except Exception:
                 pass
 
-            cmd = 'from {} import main; main(%d)'.format(main.__module__)
+            cmd = 'from {} import main; main(%d, %d)'.format(main.__module__,
+                                                             int(VERBOSE))
```

Comment:
You should remove `int(VERBOSE)` as it is not passed to the `format` API or changed the second `%d` in `{}`.
I am in favor of the later. Could you also pass the value of `r` here to have only one  formatting of command, instead of a second one in line 108?